Scripts/script-ADGetUser.yml

```diff
@@ -189,7 +189,12 @@ script: |-
   else:
       resp = listAll(attrs, demisto.get(demisto.args(),'using'), userAccountControlOut)
 
+  expectedErrs = ['invalid attribute type dn', 'Unsupported Command']
```

Comment:
expectedErrs => expected_errs
errData => err_data

Reply:
used the same style as appears on the script...